src/components/Search/UserList.vue

```diff
@@ -0,0 +1,36 @@
+<script lang="ts" setup>
+import UserListItem from './UserListItem.vue'
+import { User } from '/@/lib/apis'
+
+interface Props {
+  members: User[]
+}
+
+defineProps<Props>()
+</script>
+
+<template>
+  <div :class="$style.container">
```

Comment:
member-listの方はこういう表記になっていたんですが、ここでタグを変更した理由はありますか…？
```suggestion
  <section :class="$style.container">
```

Reply:
基本的にsectionタグの中(の一番上)には見出しを置いた方がいいのですが、今回特に置く見出しがなかった(すぐ上にh1があったし)のでsectionじゃなくて単純にdivにしたって感じです

Reply:
:haakusimasita:
じゃあこれでマージしちゃって大丈夫です！